src/test/java/test/domain/LottoPrizeCalculatorTest.java

```diff
@@ -0,0 +1,119 @@
+package test;
+
+import domain.Lotto;
+import domain.LottoPrizeCalculator;
+import domain.MatchReward;
+import domain.value.BonusNumber;
+import domain.value.Money;
+import domain.value.WinningNumbers;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class LottoPrizeCalculatorTest {
```

Comment:
[참고용]
메서드마다 한글로 적혀있으면 intellij에서 노란줄이 뜨게 되는데요. 클래스 위에 어노테이션 적용해주시면 노란줄이 안떠서 좋아요

```suggestion
@SuppressWarnings("NonAsciiCharacters")
@DisplayNameGeneration(ReplaceUnderscores.class)
class LottoPrizeCalculatorTest {
```

Reply:
노란줄이 거슬렸는데 없애는 법이 있었군요!
적용해봤는데 신기하네요... 감사합니다!